changelog/1.20.0.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.20.0"
+description: "Released on 06/16/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.20.0.
+
+### Features ✨
+
+- web: A site-wide workspace template policy may now be managed at **Manage >
+  Admin > Templates > Template Policy**. If not set, a sensible default is
+  provided.
+- web: The _nodeSelectors_ field was added to workspace templates
+- other: There is a new JsonSchema for writing Coder workspaces as code template
+  with intellisense
```

Comment:
Needs documentation (where and how to obtain and use)

cc @Emyrk

Reply:
https://github.com/cdr/docs/pull/399

Reply:
@vapurrmaid use the phrasing "Code Completion". I was made aware that "Intellisense" is trade marked by Microsoft.

Reply:
```suggestion
- other: There is a new JsonSchema for writing Coder workspaces as code template
  with code completion
```